crates/aptos/src/move_tool/mod.rs

```diff
@@ -2122,6 +2125,43 @@ impl CliCommand<TransactionSummary> for RunFunction {
     }
 }
 
+/// BETA: Simulate a Move function or script
+///
+/// BETA: subject to change
+///
+/// This allows you to simulate and see the output from any function or Move script all in one command.
+/// It additionally lets you simulate for any account
+///
+/// TODO: This should be simpler than the rest of the commands, soon to move other commands to a flow like this
+#[derive(Parser)]
+pub struct Simulate {
+    #[clap(flatten)]
+    txn_options: TxnOptions,
+    // TODO: Mix entry function and script together with some smarts
+    #[clap(flatten)]
+    entry_function_args: EntryFunctionArguments,
+
+    #[clap(long)]
```

Comment:
Doc comment here explaining what this does would be good, particularly given you might need to manually set max gas / gas unit price.